nodeEditor/src/sharedComponents/sliderLineComponent.tsx

```diff
@@ -100,14 +103,26 @@ export class SliderLineComponent extends React.Component<ISliderLineComponentPro
     }
 
     render() {
+
+        const input = this.props.globalState ?
+        <FloatLineComponent globalState={this.props.globalState} smallUI={true} label="" target={this.state} propertyName="value" min={this.prepareDataToRead(this.props.minimum)} max={this.prepareDataToRead(this.props.maximum)}

```

Comment:
why not having the input all the time?

Reply:
do you think we should? I'm not sure all the places that include it are wanted but I can check.

Reply:
I think so unless we find an obvious reason not to do so (same for inspector by the way)